### ISSUE TITLE
Debian Packaging Issues

### ISSUE THREAD

**Issue by creichert** [snoyberg/keter#70]
I am very close to finishing the debian packages but I have a few issues I would like to get opinions on.

Currently, keter has a configuration parameter `root` which sets up the target directories for several folders: `/incoming`, `/log` and `/temp`.

The current FPM packaging seems to set root to `/var/www`.  I think this can be refined a bit to fit the filesystem heirarchy standard. The debian FTP will not accept this package if this folder is re-purposed for all of these directories (https://lintian.debian.org/tags/dir-or-file-in-var-www.html).

Should these directories be configurable individually? Or, should they simply be modified directly in the source to fit a root filesystem suitable for `apt-get` install? I can patch this in Keter directly or in the Debian packaging but I want to find something that works for everyone without messing up current installs.

Basically, I think need to either separate the current directories to a suitable location or find a suitable root where they can all be installed.

```
/temp -> /tmp    (/var/www/keter/...)

/incoming -> /var/cache/keter/incoming (not so sure about this one)

/log -> /var/log/keter
```

or, create root as `/var/lib/keter` and install everything there.

```
/temp -> /var/lib/keter/incoming/temp
/incoming -> /var/lib/keter/incoming
/log -> /var/lib/keter/log   (should probably go to /var/log anyways)
```

One other issue is where to write the `postgres.yaml` file. Currently, I am planning to write both config files to `/etc/keter/keter.conf` and `/etc/keter/postgres.conf` and `chmod 600 /etc/keter/postgres.conf` at install time.

**Comment by creichert** [comment]
A lot of this was discussed here: https://groups.google.com/forum/#!searchin/yesodweb/debian/yesodweb/BeBU8V90-G8/wsaTyTLUXkUJ

It looks like the consensus was that there are no strong objections to any install locations. I originally installed to `/usr/bin` over `/usr/sbin` simply because the debian packaging uses it but it looks like `/usr/sbin` will be preferred.

**Comment by pharpend** [comment]
While we're at it, I forked the setup-keter.sh script to work on Debian: <https://github.com/pharpend/keter-debian>. I haven't checked, but I don't think the current script will work on versions of Ubuntu that use systemd, including 16.04. The thing I wrote has a systemd service file included, so that might be otherwise useful.

**Comment by ivanperez-keera** [comment]
Hi! Any progress on this?

**Comment by ivanperez-keera** [comment]
A recent commit https://github.com/snoyberg/keter/commit/d25fe8125963f2b30de31f003844091b5b7b8ed8 addressed moving from upstart to systemd.

**Comment by jezen** [comment]
I'll close this as it's clearly abandoned.
